Misc/NEWS.d/next/Library/2019-09-15-21-31-18.bpo-37828.gLLDX7.rst

```diff
@@ -0,0 +1 @@
+Properly report the mock name in :meth:`unittest.mock.assert_called` error
```

Comment:
I think this needs to be ``` :meth:`unittest.mock.Mock.assert_called` ```, as the method belongs to the Mock class.

You can test this out running `make html` in the `Docs` folder and seeing the output.

Reply:
As a nitpick, you might want to be more precise on what is fixed, so people can quickly see the fix in the "what's new" section.

Something like:
```
Fix incorrect mock name being generated in :meth:`unittest.mock.assert_called` exceptions
for mocks without a name. Patch by Abraham Toriz Cruz.
```